app/assets/stylesheets/modules/forms.scss

```diff
@@ -1,4 +1,13 @@
 .form-group {
+  a.btn:focus,
```

Comment:
@jkeck Added scss-mixin for using the outline style outside `.form-group` elements